app/presenters/claim_presenter.rb

```diff
@@ -1,6 +1,19 @@
 class ClaimPresenter < BasePresenter
   presents :claim
 
+
+  # returns a hash of state as a symbol, and state as a human readable name suitable for use in drop down
+  #
+  def valid_transitions
+    states = claim.state_transitions.map(&:to_name) - [:archived_pending_delete]
```

Comment:
do we need to remove submitted aswell? i pulled this branch and tried to set status to submitted and it blew up
<img width="1433" alt="screen shot 2015-10-15 at 13 05 29" src="https://cloud.githubusercontent.com/assets/7016425/10513142/7a5ef710-733d-11e5-8475-f8c432b5bfb5.png">